File: src/main/java/com/uwetrottmann/trakt5/entities/BaseEntity.java
package com.uwetrottmann.trakt5.entities;

import org.threeten.bp.OffsetDateTime;

import java.util.List;

public abstract class BaseEntity {

    public String title;

    // extended info
    public String overview;
    public Double rating;
    public Integer votes;
    public OffsetDateTime updated_at;
    public List<String> available_translations;

}

File: src/main/kotlin/com/redissi/trakt/entities/BaseEntity.kt
package com.redissi.trakt.entities

import com.squareup.moshi.Json
import java.time.OffsetDateTime

abstract class BaseEntity {

    abstract val title: String?
    // extended info
    abstract val overview: String?
    abstract val rating: Double?
    abstract val votes: Int?
    @Json(name = "updated_at")
    abstract val updatedAt: OffsetDateTime?
    @Json(name = "available_translations")
    abstract val availableTranslations: List<String>?

}